src/test/java/uk/gov/gchq/palisade/policy/WrappedRuleTest.java

```diff
@@ -26,10 +26,17 @@
 
     @Test
     public void shouldConstruct1ArgumentWithNoErrors() throws Exception {
-        new WrappedRule<>(null, null, null);
-        new WrappedRule<>(new TestRule(), null, null);
-        new WrappedRule<>(null, o -> o.toString(), null);
-        new WrappedRule<>(null, null, o -> true);
+        // Given
+        WrappedRule rule1 = new WrappedRule<>(null, null, null);
+        WrappedRule rule2 = new WrappedRule<>(new TestRule(), null, null);
+        WrappedRule rule3 = new WrappedRule<>(null, o -> o.toString(), null);
+        WrappedRule rule4 = new WrappedRule<>(null, null, o -> true);
+
+        // Then
+        Assert.assertNotNull(rule1);
```

Comment:
rule1 should be failing and throwing an error similar to other test cases

Reply:
The nullCount within the WarppedRule class checks if it's less than 2 so rule1 currently does not error. I can change this so that the nullCount checks for only 1 non-null parameter.